apis.py

```diff
@@ -494,191 +492,275 @@ def update_volunteer_info(*args,**kwargs):
     source = request.form.get('source')
     status = request.form.get('status')
     country = request.form.get('country')
-    req_dict = {'name':name,'mob_number':mob_number,'email_id':email_id,
-                'country':country,'address':address,'geoaddress':geoaddress,'latitude':latitude, 'longitude':longitude,
-                'source':source,'status':status}
+    req_dict = {'name': name, 'mob_number': mob_number, 'email_id': email_id,
+                'country': country, 'address': address, 'geoaddress': geoaddress, 'latitude': latitude,
+                'longitude': longitude,
+                'source': source, 'status': status}
     v_df = volunteer_data_by_id(v_id)
-    if(v_df.shape[0]==0):
-        return json.dumps({'status':False,'string_response':'Volunteer does not exist','Response':{}})
+    if (v_df.shape[0] == 0):
+        return json.dumps({'status': False, 'string_response': 'Volunteer does not exist', 'Response': {}})
     if (v_id is None):
-        return {'Response':{},'status':False,'string_response':'Volunteer ID mandatory'}
-    v_dict = {x:req_dict[x] for x in req_dict if req_dict[x] is not None}
-    response = json.dumps(update_volunteers_db({'id':v_id},v_dict))
+        return {'Response': {}, 'status': False, 'string_response': 'Volunteer ID mandatory'}
+    v_dict = {x: req_dict[x] for x in req_dict if req_dict[x] is not None}
+    response = json.dumps(update_volunteers_db({'id': v_id}, v_dict))
     return response
 
 
 # In[ ]:
 
 
-@app.route('/logout',methods=['POST'])
+@app.route('/logout', methods=['POST'])
 @login_required
-def logout_request(*args,**kwargs):
+def logout_request(*args, **kwargs):
     token = request.headers['Authorization'].split(" ")[1]
     success = blacklist_token(token)
     message = 'Logged out successfully' if success else 'Failed to logout'
-    return json.dumps({'Response': {}, 'status': success, 'string_response': message}) 
+    return json.dumps({'Response': {}, 'status': success, 'string_response': message})
 
 
 # In[ ]:
 
 
 # get requests that are available to be assigned to volunteers
-@app.route('/accept_page',methods=['GET'])
+@app.route('/accept_page', methods=['GET'])
 def request_accept_page():
     uuid = request.args.get('uuid')
     df = accept_request_page(uuid)
-    if(df.shape[0]==0):
-        return json.dumps({'Response':{},'status':False,'string_response':'This page does not exist. Redirecting to homepage'})
+    if (df.shape[0] == 0):
+        return json.dumps(
+            {'Response': {}, 'status': False, 'string_response': 'This page does not exist. Redirecting to homepage'})
     else:
-        return json.dumps({'Response':df.to_dict('records'),'status':True,'string_response':'Request related data extracted'})
+        return json.dumps(
+            {'Response': df.to_dict('records'), 'status': True, 'string_response': 'Request related data extracted'})
 
 
 # In[ ]:
+@app.route('/ngo_request_form', methods=['POST'])
```

Comment:
since we have `/create_request` for a regular request, we could perhaps name it something like `/create_ngo_request`